packages/react/runtime/__test__/list.test.jsx

```diff
@@ -2015,3 +2015,294 @@ describe('list-item JSXSpread', () => {
     `);
   });
 });
+
+describe('list componentAtIndexes', () => {
+  const s0 = __SNAPSHOT__(
+    <view>
+      <text>111</text>
+      <list id='list'>{HOLE}</list>
+    </view>,
+  );
+
+  const s1 = __SNAPSHOT__(
+    <list-item item-key={HOLE}>
+      <text>World</text>
+    </list-item>,
+  );
+
+  it('basic componentAtIndexes with async flush', () => {
+    const b = new SnapshotInstance(s0);
+    b.ensureElements();
+    const listRef = b.__elements[3];
+
+    const d0 = new SnapshotInstance(s1);
+    const d1 = new SnapshotInstance(s1);
+    const d2 = new SnapshotInstance(s1);
+    d0.setAttribute(0, { 'item-key': 'list-item-0' });
+    d1.setAttribute(0, { 'item-key': 'list-item-1' });
+    d2.setAttribute(0, { 'item-key': 'list-item-2' });
+    b.insertBefore(d0);
+    b.insertBefore(d1);
+    b.insertBefore(d2);
+    __pendingListUpdates.flush();
+
+    const fn = vi.fn();
+    const __original = globalThis.__FlushElementTree;
+    globalThis.__FlushElementTree = (_, options = {}) => {
+      const { listID, triggerLayout, operationIDs, elementIDs, asyncFlush } = options;
+      if (asyncFlush !== undefined) {
+        fn(asyncFlush);
```

Comment:
It seems that when `asyncFlush` is true, the other parameters are not tested - they do not appear in the test snapshot...